src/analysis/cytoscape/py4cy/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.9.13
+RUN pip install py4cytoscape==1.3.0
+
+# PARAMETERS
```

Comment:
Is some of this taken from the Cytoscape example?  Do we need to provide attribution?